app/web/src/newhotness/Explore.vue

```diff
@@ -927,24 +957,64 @@ const storeViewMode = () => {
 };
 
 // Group By
-const groupBySelection = ref("");
-const groupByOptions = [
-  { value: "Diff Status", label: "Diff Status" },
-  { value: "Qualification Status", label: "Qualification Status" },
-  { value: "Upgradeable", label: "Upgradeable" },
+enum GroupByCriteria {
+  Diff = "Diff Status",
+  Upgrade = "Upgradeable",
+  Qualification = "Qualification Status",
+  None = "",
+}
+
+const groupByFromString = (s: string): GroupByCriteria => {
+  const key = (
+    _.keys(GroupByCriteria) as (keyof typeof GroupByCriteria)[]
+  ).find((k) => GroupByCriteria[k] === s);
+
+  if (!key) return GroupByCriteria.None;
+  else return GroupByCriteria[key];
+};
+
+const groupBySelection = ref<GroupByCriteria>(GroupByCriteria.None);
+const groupByDropDownOptions = [
+  { value: GroupByCriteria.Diff, label: "Diff Status" },
+  { value: GroupByCriteria.Qualification, label: "Qualification Status" },
+  { value: GroupByCriteria.Upgrade, label: "Upgradeable" },
 ];
 
+// Update the hash of the route (allowing for URL links) when the group by selection changes.
+watch([groupBySelection], () => {
+  const query: SelectionsInQueryString = {
+    ...router.currentRoute.value?.query,
+  };
+  delete query.map;
+  delete query.groupBy;
+
+  query.grid = "1";
+  router.push({ query });
+
+  if (groupBySelection.value === GroupByCriteria.Diff) {
+    query.groupBy = "diff";
+  } else if (groupBySelection.value === GroupByCriteria.Qualification) {
+    query.groupBy = "qual";
+  } else if (groupBySelection.value === GroupByCriteria.Upgrade) {
+    query.groupBy = "upgr";
+  }
+
+  router.push({
+    query,
+  });
+});
+
 const getQualificationStatusTitle = (component: ComponentInList) => {
   const status = getQualificationStatus(component);
   switch (status) {
     case "success":
-      return "Passed";
+      return "Passed qualifications";
     case "running":
-      return "Running";
+      return "Running qualifications";
```

Comment:
"Running" is not real and will die in a follow up PR. This is from before qualifications were on the prop tree... might be like... since late 2022 😅. For now, we'll keep it since we discovered this right as we were wrapping it. It is harmless.